targets/TARGET_ONSEMI/TARGET_NCS36510/gpio_irq_api.c

```diff
@@ -157,10 +161,8 @@ int gpio_irq_init(gpio_irq_t *obj, PinName pin, gpio_irq_handler handler, uint32
      * then change this setting to  obj->GPIOMEMBASE->W_IN |= obj->pinMask. All parameter setting needs to change from = to |=
      */
```

Comment:
I have tested with two buttons as well as the same button triggering two inputs. Both scenarios are working.